.github/workflows/api-tests.yaml

```diff
@@ -115,92 +115,83 @@ jobs:
 
       - name: Run Users API tests
         if: steps.changes.outputs.users == 'true'
-        uses: schemathesis/action@v1
+        uses: schemathesis/action@v2.0.0
         with:
           schema: apidocs/openapi/users.yaml
           base-url: ${{ env.USERS_URL }}
           checks: all
-          report: false
           args: '--header "Authorization: Bearer ${{ env.USER_TOKEN }}" --contrib-openapi-formats-uuid --hypothesis-suppress-health-check=filter_too_much --stateful=links'
 
       - name: Run Groups API tests
         if: steps.changes.outputs.groups == 'true'
-        uses: schemathesis/action@v1
+        uses: schemathesis/action@v2.0.0
         with:
           schema: apidocs/openapi/groups.yaml
           base-url: ${{ env.GROUPS_URL }}
           checks: all
-          report: false
           args: '--header "Authorization: Bearer ${{ env.USER_TOKEN }}" --contrib-openapi-formats-uuid --hypothesis-suppress-health-check=filter_too_much --stateful=links'
 
       - name: Run Clients API tests
         if: steps.changes.outputs.clients == 'true'
-        uses: schemathesis/action@v1
+        uses: schemathesis/action@v2.0.0
         with:
           schema: apidocs/openapi/clients.yaml
           base-url: ${{ env.CLIENTS_URL }}
           checks: all
-          report: false
           args: '--header "Authorization: Bearer ${{ env.USER_TOKEN }}" --contrib-openapi-formats-uuid --hypothesis-suppress-health-check=filter_too_much --stateful=links'
 
       - name: Run Channels API tests
         if: steps.changes.outputs.channels == 'true'
-        uses: schemathesis/action@v1
+        uses: schemathesis/action@v2.0.0
         with:
           schema: apidocs/openapi/channels.yaml
           base-url: ${{ env.CHANNELS_URL }}
           checks: all
-          report: false
           args: '--header "Authorization: Bearer ${{ env.USER_TOKEN }}" --contrib-openapi-formats-uuid --hypothesis-suppress-health-check=filter_too_much --stateful=links'
 
       - name: Run HTTP Adapter API tests
         if: steps.changes.outputs.http == 'true'
-        uses: schemathesis/action@v1
+        uses: schemathesis/action@v2.0.0
         with:
           schema: apidocs/openapi/http.yaml
           base-url: ${{ env.HTTP_ADAPTER_URL }}
           checks: all
-          report: false
           args: '--header "Authorization: Client ${{ env.CLIENT_SECRET }}" --contrib-openapi-formats-uuid --hypothesis-suppress-health-check=filter_too_much --stateful=links'
```

Comment:
There were a few more changes in the latest release:

`--hypothesis-suppress-health-check` -> `--suppress-health-check`

These two were removed:

- `--contrib-openapi-formats-uuid` - now it is a built-in
- `--stateful`. All testing phases are enabled by default, and could be tuned with `--phases`.